src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Color;
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int bottom;
+    private final int up;
+    private final int height;
+
+    public IsoscelesTrapezoid(int bottom, int up, int height, Color color) {
+        this.color = color;
+        this.up = up;
+        this.height = height;
+        this.bottom = bottom;
+    }
+    @Override
+    public double countArea() {
+        return (height+bottom) / 2.0 * height;
```

Comment:
The formula for calculating the area of an isosceles trapezoid is incorrect. It should be `((bottom + up) / 2.0) * height` to ensure accurate calculations.